test/stripe/list_object_test.rb

```diff
@@ -25,20 +25,80 @@ class ListObjectTest < Test::Unit::TestCase
       assert_equal expected, list.each.to_a
     end
 
-    should "provide #auto_paging_each" do
+    should "provide #reverse_each" do
       arr = [
         { id: 1 },
         { id: 2 },
         { id: 3 },
       ]
+      expected = Util.convert_to_stripe_object(arr.reverse, {})
+      list = Stripe::ListObject.construct_from(data: arr)
+      assert_equal expected, list.reverse_each.to_a
+    end
+
+    should "provide #auto_paging_each that supports forward pagination" do
```

Comment:
This is the existing test for forward pagination, but I fleshed it out a little more to paginate across multiple pages for a little better certainty that the feature works.